### PR TITLE
[TW-925] Doc annotations support in API builder

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Updated "Commenting on APIs" section
- Added resolving comments
- Added commenting on Git-linked APIs
- Updated screenshots

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/managing-apis.md [hunk 1]
```diff
@@ -80,27 +80,65 @@ You will also receive an email with the information regarding who made the chang
 
 ## Commenting on APIs
 
-You can discuss your APIs with collaborators in Postman, keeping the conversation in context and available to other stakeholders. Anyone with access to the API can comment.
+You can discuss your APIs with collaborators in Postman, keeping the conversation in context and available to other stakeholders. Anyone with access to the API can comment. You can comment on an API, or you can leave inline comments on an API definition.
 
-To leave a comment on an API, select an API in the sidebar to expand it, then select **Definition**. Select an API definition file in the sidebar to open it, then switch to comment mode by selecting the comment icon <img alt="Comments icon" src="https://assets.postman.com/postman-docs/icon-comments-v9.jpg#icon" width="18px"> in the editor pane. To leave an inline comment, highlight part of the definition, enter your comment, and select **Add Comment**.
+To comment on an API:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/managing-apis.md:85]:
```suggestion
To comment on an API, do the following:
```

File: src/pages/docs/designing-and-developing-your-api/managing-apis.md [hunk 2]
```diff
@@ -80,27 +80,65 @@ You will also receive an email with the information regarding who made the chang
...
+1. Select an API in the sidebar.
+1. Select the comment icon <img alt="Comments icon" src="https://assets.postman.com/postman-docs/icon-comments-v9.jpg#icon" width="18px"> in the right sidebar.
+1. If there are existing comments, select **Add Comment** to add a new comment.
+1. Enter your comment and select **Add Comment**.
 
-<img alt="Commenting on an API" src="https://assets.postman.com/postman-docs/v10/api-builder-add-comment-v10.jpg" />
+<img alt="Commenting on an API" src="https://assets.postman.com/postman-docs/v10/api-builder-add-comment-v10-10.jpg" width="443px" />
 
-### Formatting comments
+To leave an inline comment on an API definition:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/managing-apis.md:94]:
```suggestion
To leave an inline comment on an API definition, do the following:
```

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good! I added a few suggestions for procedure lead-in sentences. Thanks!

**Review by petewitcher-postman — APPROVED** [review]
- LGTM! 😁  

Converting that paragraph into a procedure is a huge improvement.

File: src/pages/docs/designing-and-developing-your-api/managing-apis.md [hunk 3]
```diff
@@ -80,27 +80,77 @@ You will also receive an email with the information regarding who made the chang
...
+> To make sure you're viewing the latest comments, select the refresh icon <img alt="Refresh icon" src="https://assets.postman.com/postman-docs/icon-refresh-v9-5.jpg#icon" width="14px"> in the comment box.
 
-[![API comment manage](https://assets.postman.com/postman-docs/v8-managing-comment-actions2.jpg)](https://assets.postman.com/postman-docs/v8-managing-comment-actions2.jpg)
+### Formatting API comments
 
-> With Admin permissions you can delete comments made by any contributor, but you can't update or change comments.
+You can use Markdown in your comments. Check out the [Markdown Cheatsheet on GitHub](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for tips on formatting.
+
+To mention a collaborator in a comment, type `@` followed by their name, then choose the person from the auto-suggested list. When you mention a member in a comment, Postman sends them a notification.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/managing-apis.md:110]:
```suggestion
To mention a collaborator in a comment, type `@` followed by their name, then choose the person from the auto-suggested list. When you mention a user in a comment, Postman sends them a notification.
```

**Review by jkonrath-postman — APPROVED** [review]
One suggestion, but otherwise LGTM!

File: src/pages/docs/designing-and-developing-your-api/managing-apis.md [hunk 4]
```diff
@@ -80,27 +80,77 @@ You will also receive an email with the information regarding who made the chang
 
 ## Commenting on APIs
 
-You can discuss your APIs with collaborators in Postman, keeping the conversation in context and available to other stakeholders. Anyone with access to the API can comment.
+You can discuss your APIs with collaborators in Postman, keeping the conversation in context and available to other stakeholders. Anyone with access to the API can comment. You can comment on an API, or you can leave inline comments on an API definition.
```
Comment (ramji367) [inline src/pages/docs/designing-and-developing-your-api/managing-apis.md:83]: _Anyone with access to the API can comment_

Should we call out the fact that only people with edit access on the API can see this comment? People without edit access can only see published versions. If you think this makes it more confusing let's leave it as-is.

Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/managing-apis.md]: @ramji367 Yes, we should definitely call this out. I updated the text to clarify that editors can leave comments on APIs, but viewers can't see those comments and can only leave comments on published API versions

**Pull request merged**